qiskit/optimization/converters/ising_to_quadratic_program.py

```diff
@@ -27,13 +27,16 @@
 class IsingToQuadraticProgram:
     """Convert a qubit operator into a quadratic program"""
 
-    def __init__(self) -> None:
+    def __init__(self, linear: bool = False) -> None:
         """Initialize the internal data structure."""
```

Comment:
need to add docstring for `linear`

Reply:
oops, Alright. I added doctoring for `linear`